docs/wiki/[User-Guide]-Starter-Module-HubNetworking.md

```diff
@@ -19,7 +19,7 @@ The `hubnetworking` module is used to deploy connectivity resources such as Virt
 ### `vnet-gateway`
 
 The `vnet-gateway` module is used to deploy a Virtual Network Gateway inside your Virtual Network. By default, the resources of the module will not be deployed unless `virtual_network_gateway_creation_enabled` is set to true, if so, the module will deploy a VPN Gateway with SKU VpnGw1.
```

Comment:
In the complete file we use `virtual_network_gateway` as opposed to the term `vnet_gateway`. Just checking if that is correct / consistent?

Reply:
Yes good shout, I'm gonna rename any refs to avm-ptn-vnetgateway across the docs.